packages/kubernetes-1.23/static-pods-manifest

```diff
@@ -0,0 +1,12 @@
+[required-extensions]
+kubernetes = "v1"
+std = { version = "v1", helpers = ["if_not_null", "base64_decode"]}
++++
+{{#if_not_null settings.kubernetes.static-pods}}
+{{#each settings.kubernetes.static-pods}}
+{{#if this.enabled}}
+{{base64_decode this.manifest}}
+---
```

Comment:
Sorry for the dumb question: why is this `---` here?

Reply:
--- is a separator for deliminiting multiple pods in a kubernetes manifest, so its needed to delimit each manifest in the file, kubernetes is also okay with a trailing --- (won't error on expecting a second)